source/decoupled_authentication.rst

```diff
@@ -0,0 +1,125 @@
+.. _3ds_decoupled_authentication:
+
+Decoupled Authentication
+=========================
+
+There are four different authentication flows in the 3-D Secure protocol. The decoupled authentication
+flow is one of them. The decoupled authentication flow is an asynchronous process.
+
+The ACS can determine decoupled authentication fallback for an authentication transaction.
+If a fallback is determined, `threeDSRequestorPriorAuthenticationInfo` is required.
+
+The requestor can set the TTL for the transaction by setting the `threeDSRequestorDecMaxTime`.
+The `threeDSRequestorDecMaxTime` accepts numeric values between `00001` and `10080` (7 days) which
+determines the TTL in minutes.
+
+The requestor shall use the `threeDSServerTransID` to retrieve the authentication value
+by making a request towards the `/postauth` endpoint.
+Once we receive the authentication value from the ACS, we will only store the authentication value for 20 minutes.
+We suggest that the requestor should poll the `/postauth` endpoint at least every 20 minutes.
+If the requestor does not poll the `/postauth` endpoint within the TTL, the authentication value will be deleted.
```

Comment:
I guess we always deleted the AV after the TTL has passed, not only if the requestor has not polled for it :wink: 

```suggestion
The authentication value is deleted when the TTL has passed.
```

:man_shrugging: